sphinx_qt_documentation/__init__.py

```diff
@@ -6,45 +6,41 @@
 `qt_documentation` with possible values:
 
  * PyQt5 - linking to PyQt documentation on https://www.riverbankcomputing.com/static/Docs/PyQt5/api/ (incomplete)
- * Qt5 - linking to Qt5 documentation on "https://doc.qt.io/qt-5/" (default)
- * PySide2 - linking to PySide2 documentation on  "https://doc.qt.io/qtforpython/PySide2/"
+ * Qt5 - linking to Qt5 documentation on "https://doc.qt.io/qt-5/"
+ * PySide2 - linking to PySide6 documentation on  "https://doc.qt.io/qtforpython-5/"
+ * PyQt6 - linking to PyQt documentation on https://www.riverbankcomputing.com/static/Docs/PyQt6/api/ (incomplete)
+ * Qt6 - linking to Qt5 documentation on "https://doc.qt.io/qt-6/" (default)
+ * PySide6 - linking to PySide6 documentation on  "https://doc.qt.io/qtforpython/PySide6/"
 """
 from typing import Any, Dict
 
 from sphinx.application import Sphinx
 from sphinx.config import ENUM
 
-from sphinx_qt_documentation._missing_reference import (
-    _get_signal_and_version,
+from sphinx_qt_documentation.utils import (
     autodoc_process_signature,
     missing_reference,
+    patch_intersphinx_mapping,
 )
 
 
 def setup(app: Sphinx) -> Dict[str, Any]:
     app.setup_extension("sphinx.ext.autodoc")
     app.setup_extension("sphinx.ext.intersphinx")
     app.add_config_value(
-        "qt_documentation", "Qt5", True, ENUM("Qt5", "PySide2", "PyQt5")
+        "qt_documentation",
+        "Qt6",
```

Comment:
```suggestion
        "Qt5",
```

Qt6 is not yet the default choice for many projects because not all modules are available yet. This will need Qt 6.4 or even 6.5.